### PR TITLE
Create separate pages for querying data

### PR DESCRIPTION
**Pull request by kelseiv**
Addresses the following issues in https://github.com/influxdata/docs-v2/issues/1688:
- [x] On the "Query with the InfluxDB API" page, 
       - Link to actual API doc for Query InfluxDB `/api/v2/query` endpoint
       - Provide query InfluxQL examples

- [x] Divvy up content from the "Execute queries" page into separate pages to improve accessibility in Google search results. 

- [x] Add the following work to the influxdb repo, where this content is sourced and maintained:
        - [Add preface or introductory material](https://github.com/influxdata/influxdb/issues/19915) at the beginning of the swagger API docs. 
        - Looks like the "Query" TOC results sort order has been updated "Query InfluxDB"  appears first and "PostQueryAst" last.
- [ ] Duplicate new pages in the Cloud directory for Cloud/OSS doc split

### CODE REVIEW

File: content/influxdb/v2.0/query-data/execute-queries/data-explorer.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+menu:
+  influxdb_2_0:
+    name: Query with Data Explorer
+    parent: Execute queries
+influxdb/v2.0/tags: [query]
+---
+
+Queries can be built, executed, and visualized in InfluxDB UI's Data Explorer.
+
+![Data Explorer with Flux](/img/influxdb/2-0-data-explorer.png)
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:13]: I'd walk through the basics of building a query with the Data Explorer, then provide this link: 

https://docs.influxdata.com/influxdb/v2.0/visualize-data/explore-metrics/#explore-data-with-flux-and-the-data-explorer

I goes into more detail about using the Data Explorer.

File: content/influxdb/v2.0/query-data/execute-queries/flux-repl.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    parent: Execute queries
+influxdb/v2.0/tags: [query]
+---
+
+The [Flux REPL](/influxdb/v2.0/tools/repl/) starts an interactive
+Read-Eval-Print Loop (REPL) where you can write and execute Flux queries.
+
+```sh
+./flux repl
+```
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/flux-repl.md:16]: I expect some further details in this doc, but I think that expectation can be met with a link to the REPL doc _after_ the code block. What about this:

~~~md
The **Flux REPL** starts an interactive Read-Eval-Print Loop (REPL) where you can write and execute Flux queries.

```sh
./flux repl
```

For more information, see [Use the Flux REPL](/influxdb/v2.0/tools/repl/).
~~~

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 3]
```diff
@@ -1,54 +1,13 @@
...
+title: Query with the InfluxDB API
+weight: 202
 menu:
   influxdb_2_0:
-    name: Execute queries
-    parent: Query data
+    name: Query with the InfluxDB API
+    parent: Execute queries
 influxdb/v2.0/tags: [query]
 ---
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:10]: General comment for this doc – we may want to introduce the `/query` compatiblity endpoint in this doc if users want to query with InfluxQL.

File: content/influxdb/v2.0/query-data/execute-queries/influx-query.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: Use the `influx query` command
+weight: 204
+menu:
+  influxdb_2_0:
+    name: Use the `influx query` command
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-query.md:6]: - Don't use inline code formatting in menu item names.
- You could shorten this to "Use the influx CLI" since this only appears in the left nav.

File: content/influxdb/v2.0/query-data/execute-queries/influx-query.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    parent: Execute queries
+influxdb/v2.0/tags: [query]
+---
+
+## Influx query command
+
+You can pass queries to the [`influx query` command](/influxdb/v2.0/reference/cli/influx/query)
+as either a file or raw Flux via stdin.
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-query.md:14]:
```
Use the [`influx query` command](/influxdb/v2.0/reference/cli/influx/query) to query data in InfluxDB using Flux.
Pass Flux queries to the command as either a file or via stdin.
```

File: content/influxdb/v2.0/query-data/execute-queries/influx-query.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: Use the `influx query` command
+weight: 204
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-query.md:3]: This page needs a description.

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 7]
```diff
@@ -1,54 +1,13 @@
...
-title: Execute queries
-seotitle: Different ways to query InfluxDB
-description: There are multiple ways to query data from InfluxDB including the InfluxDB UI, CLI, and API.
-weight: 103
+title: Query with the InfluxDB API
+weight: 202
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:3]: This page needs a description.

File: content/influxdb/v2.0/query-data/execute-queries/flux-repl.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+---
+title: Query in the Flux REPL
+weight: 203
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/flux-repl.md:3]: This page needs a description.

File: content/influxdb/v2.0/query-data/execute-queries/data-explorer.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+---
+title: Query in Data Explorer
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:2]: This page needs a description.

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.0/query-data/execute-queries/data-explorer.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:13]: Hmm, this looks like the same procedure?
To simplify, prob makes sense to retitle "Explore metrics with InfluxDB" to "Query data in Data Explorer" and move over to Query data....

File: content/influxdb/v2.0/query-data/execute-queries/flux-repl.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/v2.0/query-data/execute-queries/flux-repl.md:16]: Notice you dropped "Interactive" here, shall we drop from the heading?

File: content/influxdb/v2.0/query-data/execute-queries/data-explorer.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:13]: I'd be fine with that. Just be sure to add the alias so any links point to the new location.

File: content/influxdb/v2.0/query-data/execute-queries/flux-repl.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/v2.0/query-data/execute-queries/flux-repl.md:16]: Oh, that wasn't intentional 😄 . You can leave it in.

File: content/influxdb/v2.0/query-data/execute-queries/data-explorer.md [hunk 10]
```diff
@@ -10,20 +12,14 @@ menu:
 influxdb/v2.0/tags: [query]
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:12]: Add the following to the frontmatter:

```yml
aliases:
  - /influxdb/v2.0/visualize-data/explore-metrics/
```

Comment (kelseiv) [reply content/influxdb/v2.0/query-data/execute-queries/data-explorer.md:12]: line 6 won't work? (copied and pasted format from another page, so want to be sure)

**Comment by sanderson** [comment]
@kelseiv Quick reminder – duplicate these new pages in the cloud section. Some will be able to use the `{{< duplicate-oss >}}` shortcode for the content, but some may not.

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+- Query in Flux or InfluxQL with the request's raw data.
+
+{{% note %}}
+#### Use gzip to compress the query response
+
+To compress the query response, set the `Accept-Encoding` header to `gzip`.
+This saves network bandwidth, but increases server-side load.
+{{% /note %}}
+
+#### Example query request
```
Comment (kelseiv) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:32]: Will here's the curl eg we were chatting about

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+{{% code-tab-content %}}
+```bash
+curl http://localhost:8086/api/v2/query?org=my-org -XPOST -sS \
+  --header 'Authorization: Token YOURAUTHTOKEN' \
+  --header 'Accept: application/csv' \
+  --header 'Content-type: application/vnd.flux' \
+  --data 'from(bucket:"example-bucket")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "example-measurement")
+        |> aggregateWindow(every: 1h, fn: mean)'
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:51]:
```sh
curl --request POST  \
  http://localhost:8086/api/v2/query?org=my-org  \
  --header 'Authorization: Token YOURAUTHTOKEN' \
  --header 'Accept: application/csv' \
  --header 'Content-type: application/vnd.flux' \
  --data 'from(bucket:"example-bucket")
        |> range(start: -12h)
        |> filter(fn: (r) => r._measurement == "example-measurement")
        |> aggregateWindow(every: 1h, fn: mean)'
```

or 

```sh
curl --request POST  http://localhost:8086/api/v2/query  \
  --header 'Authorization: Token YOURAUTHTOKEN' \
  --header 'Accept: application/csv' \
  --header 'Content-type: application/vnd.flux' \
  --data-urlencode 'org=my-org' \
  --data 'from(bucket:"example-bucket")
        |> range(start: -12h)
        |> filter(fn: (r) => r._measurement == "example-measurement")
        |> aggregateWindow(every: 1h, fn: mean)'
```

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+[With compression](#)
+{{% /code-tabs %}}
+
+{{% code-tab-content %}}
+```bash
+curl --request -G http://localhost:8086/query?org=my-org?database=MyDB&retention_policy=MyRP \
+  --header 'Authorization: Token YOURAUTHTOKEN' \
+  --header 'Accept: application/csv' \
+  --header 'Content-type: application/json' \
+  --data-urlencode "q=SELECT used_percent FROM example-db.example-rp.example-measurement WHERE host=host1"
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:84]: This should work:

```sh
curl --request GET \
  http://localhost:8086/query?org=my-org?database=MyDB&retention_policy=MyRP \
  --header 'Authorization: Token YOURAUTHTOKEN' \
  --header 'Accept: application/csv' \
  --header 'Content-type: application/json' \
  --data-urlencode "q=SELECT used_percent FROM example-db.example-rp.example-measurement WHERE host=host1"
```

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+{{% /code-tab-content %}}
+
+{{% code-tab-content %}}
+```bash
+curl --request -G http://localhost:8086/query?org=my-org?database=MyDB&retention_policy=MyRP \
+  --header 'Authorization: Token YOURAUTHTOKEN' \
+  --header 'Accept: application/csv' \
+  --header 'Content-type: application/json' \
+  --header 'Accept-Encoding: gzip' \
+  --data-urlencode "q=SELECT used_percent FROM example-db.example-rp.example-measurement WHERE host=host1"
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/execute-queries/influx-api.md:95]:
```js
curl --request GET \
  http://localhost:8086/query?org=my-org?database=MyDB&retention_policy=MyRP \
  --header 'Authorization: Token YOURAUTHTOKEN' \
  --header 'Accept: application/csv' \
  --header 'Content-type: application/json' \
  --header 'Accept-Encoding: gzip' \
  --data-urlencode "q=SELECT used_percent FROM example-db.example-rp.example-measurement WHERE host=host1"
```

File: content/influxdb/v2.0/query-data/execute-queries/influx-api.md (hunk 13 above)
Comment (kelseiv) [reply content/influxdb/v2.0/query-data/execute-queries/influx-api.md:84]: @sanderson here's [the issue](https://github.com/influxdata/docs-v2/issues/1641) we were discussing earlier (if it makes a difference here?)
